sktime/forecasting/base/_base.py

```diff
@@ -23,7 +23,7 @@ class name: BaseForecaster
 Inspection methods:
     hyper-parameter inspection  - get_params()
     fitted parameter inspection - get_fitted_params()
-    current ForecastingHorizon  - fh
+    current ForecastngHorizon  - fh
```

Comment:
```suggestion
    current ForecastingHorizon  - fh
```

The modification looks like a typo.